docs/userguide/constraints.md

```diff
@@ -222,10 +233,123 @@ DiscreteLinkedParametersConstraint(
 | 3 | Octanol           | Octanol             |                                         |
 
 ### ``DiscreteDependenciesConstraint``
-Content coming soon...
+A dependency is a situation where parameters depend on other parameters.
+Let's say an experimental setup has a parameter called "Switch" which turns on 
```

Comment:
Since we are speaking of explicit parameters and values here, I would prefer to have Switch and on in backticks already here.

Reply:
do you mean like `on`/`off` or more like `"on"`/`"off"` ?

Reply:
I do not care too much about that but it should be in backticks in general imo.

Reply:
ok I cahnged on/off now to have backticks, but Im not sure Switch should have them as its not an object itself or part of the possible values of an object

Reply:
ok I now went with
- using backticks when saying a parameter is called `"Switch"`
- using no backticks when speaking about it like "the switch parameter"

is that OK?

Reply:
Works for me.